selfdrive/car/fw_versions.py

```diff
@@ -328,7 +331,8 @@ def get_fw_versions(logcan, sendcan, extra=None, timeout=0.1, debug=False, progr
     for addr_chunk in chunks(addr):
       for r in REQUESTS:
         try:
-          addrs = [(a, s) for (b, a, s) in addr_chunk if b in (r.brand, 'any')]
+          addrs = [(b, a, s) for (b, a, s) in addr_chunk if b in (r.brand, 'any') and
```

Comment:
```suggestion
          addrs = [(a, s) for (b, a, s) in addr_chunk if b in (r.brand, 'any') and
```

As-written, I get a flurry of tracebacks after the VIN query, and I don't think you intended to change that tuple.

```
FW query exception: Traceback (most recent call last):                                                                
  File "./fw_versions.py", line 338, in get_fw_versions                                                               
    query = IsoTpParallelQuery(sendcan, logcan, r.bus, addrs, r.request, r.response, r.rx_offset, debug=debug)        
  File "/data/pythonpath/selfdrive/car/isotp_parallel_query.py", line 29, in __init__                                 
    self.msg_addrs = {tx_addr: get_rx_addr_for_tx_addr(tx_addr[0], rx_offset=response_offset) for tx_addr in self.real
_addrs}                                                                                                               
  File "/data/pythonpath/selfdrive/car/isotp_parallel_query.py", line 29, in <dictcomp>                               
    self.msg_addrs = {tx_addr: get_rx_addr_for_tx_addr(tx_addr[0], rx_offset=response_offset) for tx_addr in self.real
_addrs}                                                                                                               
  File "/data/pythonpath/panda/python/uds.py", line 518, in get_rx_addr_for_tx_addr                                   
    if tx_addr < 0xFFF8:                                                                                              
TypeError: '<' not supported between instances of 'str' and 'int'  
```

Reply:
Whoops, yeah that was for debugging purposes. Thanks for catching!